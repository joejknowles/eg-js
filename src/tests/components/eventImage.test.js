import Component from '../../components/eventImage';

import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

const shallowRenderer = ReactTestUtils.createRenderer();

it('renders event image as before with url', () => {
  const component = shallowRenderer.render(<Component url='something' />);
  expect(component).toMatchSnapshot();
});

it('renders event image as before without url', () => {
  const component = shallowRenderer.render(<Component />);
  expect(component).toMatchSnapshot();
});
