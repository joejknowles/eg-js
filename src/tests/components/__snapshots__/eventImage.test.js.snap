exports[`test renders event image as before with url 1`] = `
<img
  alt="event"
  className="event-image"
  src="something" />
`;

exports[`test renders event image as before without url 1`] = `
<img
  alt="event"
  className="event-image"
  src="https://placehold.it/100x100?text=IMAGE" />
`;
